optaweb-employee-rostering-frontend/.eslintrc.json

```diff
@@ -1,7 +1,7 @@
 {
   "parser": "@typescript-eslint/parser",
   "parserOptions": {
-    "project": "./tsconfig.json"
+    "project": ["./tsconfig.json", "cypress/tsconfig.json"]
```

Comment:
Why this? Do we have any TypeScript code in `cypress` directory?

Reply:
Because the cypress directory is passed to eslint, and eslint fails if it finds a file not in a project. (It just javascript files, but I need a project so eslint know the files are in the project).